### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Fedot0v**

### CODE REVIEW

File: frontend/src/views/MovieAddScreen.vue [hunk 1]
```diff
@@ -121,7 +121,7 @@ export default {
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies-${movie.id}-upload-image/`, data, imageConfig);
```
Comment (pavlopro) [inline frontend/src/views/MovieAddScreen.vue:124]: Are you sure that this works?

**Review by pavlopro — CHANGES_REQUESTED** [review]

**Review by vsmutok — CHANGES_REQUESTED** [review]
Please attach screenshots showing that the images on website are displaying correctly.

**Comment by Fedot0v** [comment]
Screenshot_10

File: frontend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /app
+
+COPY frontend/package*.json ./
+
+RUN npm install --include=dev
+
+COPY ./frontend ./
+
+CMD ["npm", "run", "dev", "--", "--host"]
```
[the remark this answers is not in the document]
Comment (Fedot0v) [reply frontend/Dockerfile]: Screenshot_11
i don't have images for movies from json file

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Please, populate these images using either API or admin panel to avoid error responses in the dev console

**Comment by Fedot0v** [comment]
Screenshot_12

**Pull request closed without merging**